openassessment/data.py

```diff
@@ -36,7 +37,7 @@ def _usernames_enabled():
     Checks if toggle for deanonymized usernames in report enabled.
     """
 
-    return settings.FEATURES.get('ENABLE_ORA_USERNAMES_ON_DATA_EXPORT', False)
+    return settings.FEATURES.get("ENABLE_ORA_USERNAMES_ON_DATA_EXPORT", False)
```

Comment:
We shouldn't add style changes to things we haven't modified, it generates unnecessary noise.